infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/DraftableObjectService.kt

```diff
@@ -38,6 +38,14 @@ abstract class DraftableObjectService<ObjectType: Draftable<ObjectType>, DaoType
         return getInternal(publishType, id)
     }
 
+    fun getMany(publishType: PublishType, ids: List<IntId<ObjectType>>): List<ObjectType> {
+        logger.serviceCall("getMany", "publishType" to publishType, "ids" to ids)
+        return when (publishType) {
+            DRAFT -> dao.fetchDraftVersionsOrThrow(ids).map { dao.fetch(it) }
```

Comment:
Tuon .map -osion voisi nostaa when-blokin ulkopuolelle. Lisäksi ainakin itse tykkään enemmän funktioref syntaksista kuin default it-parametrista:
```
when (publishType) {
  DRAFT -> ...
}.map(dao::fetch)
```

Reply:
Jeps, tykkään kanssa itse tuosta tyylistä enemmän. Korjailen tämän toisella tiketillä, missä teen vähän lisää näitä massahakuja